common/djangoapps/track/middleware.py

```diff
@@ -24,6 +24,7 @@
 log = logging.getLogger(__name__)
 
 CONTEXT_NAME = 'edx.request'
+HTTP_USER_AGENT_KEY = 'agent'
```

Comment:
@Shrhawk , from the last review cycle, it sounded like it made more sense not to include this as a global variable. Just use 'agent'. @mulby , I'm understanding you correctly here, right?